### ISSUE TITLE
API reference typo

### ISSUE THREAD

**Issue by ice1000** [Codewars/codewars.com#982]
https://dev.codewars.com/#get-completed-code-challenges-for-a-given-user

I saw:

```
curl "http://www.codewars.com:3000/api/v1/users/some_user/code_challenges/completed"
```

But the API is actually:

```
curl "http://www.codewars.com/api/v1/users/some_user/code-challenges/completed"
```

Two mistakes:

+ port number
+ it's minus but not underline

**Comment by kazk** [comment]
It's been fixed (Codewars/codewars-api-docs#6), but the documentation hasn't been rebuilt to reflect that :(

I've asked about how this is done few times, but got no response.
I also suggested (https://github.com/Codewars/codewars.com/issues/881#issuecomment-299574165) moving them into the new documentation which reflects the change automatically.

- <https://kazk.github.io/docs.codewars.com/api/>

**Comment by kazk** [comment]
The documentation has been updated.

![image](https://cloud.githubusercontent.com/assets/639336/26597917/7162b9bc-4528-11e7-9c01-ded2c30b5edd.png)

<https://dev.codewars.com/#get-user:-completed-challenges>
